### ISSUE TITLE
Files with duplicate imports results in missing computed property bodies 

### ISSUE THREAD

**Issue by omairvaiyani** [ember-codemods/ember-native-class-codemod#175]
The files in which I have a duplicated import, for .e.g

```javascript
import { set } from '@ember/object';
import { computed } from '@ember/object';
```

result in the function body from `computed` properties being lost.

Duplicated imports
```javascript
import classic from 'ember-classic-decorator';
import Service from '@ember/service';
import { set } from '@ember/object';
import { computed } from '@ember/object';

@classic
export default class ExampleService extends Service {
	init() {
          super.init(...arguments);
	  set(this, 'foo', 'bar');
	}

	@computed()
	baz;
}
```

Unique imports
```javascript
import classic from 'ember-classic-decorator';
import Service from '@ember/service';
import { set, computed } from '@ember/object';

@classic
export default class ExampleService extends Service {
	init() {
	  super.init(...arguments);
	  set(this, 'foo', 'bar');
	}

	@computed()
	get baz() {
	  return true;
	}
}
```

I plan to use some regex to find and resolve these duplicated imports before running the codemod again, but I thought it'd be useful for other people to be aware of this bug.

**Comment by pzuraq** [comment]
Hey @omairvaiyani, we've released a new version with a bunch of bugfixes, can you check to see if this is still an issue with the latest version of the codemod?

**Comment by omairvaiyani** [comment]
Resolved in `1.1.0` Nice work
